measurements-to-gelf.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python
+
+#
+# Prototype of a RIPE Atlas to GELF parser by awlx 
+#
+# Usage: python measurements-to-gelf.py <id> <min>
+#
+
+# Imports
+from gelfclient import UdpClient
+from urllib2 import urlopen
+import syslog
+import json
+import time
+import sys
+import requests
+
+# Configuration
+gelf_server = 'localhost'
+gelf_port = 5555
+api_key = '<api_key>'
+
+# Parse Args
+if len(sys.argv) == 1:
+    print "# Prototype of a RIPE Atlas to GELF parser"
```

Comment:
Python 2.6+ supports invoking `print` with parentheses (`print("this is my stuff")`).  More importantly, Python 3 enforces the use of parentheses in this case, so if you don't use them, you're preventing this code from being used on modern installations.  I'd recommend this line look like this:

```python
print("# Prototype of a RIPE Atlas to GELF parser")
```